launchdarkly_rum/README.md

```diff
@@ -0,0 +1,33 @@
+# LaunchDarkly Feature flag tracking integration
+
+## Overview
+
+LaunchDarkly's feature flag tracking integration enriches your RUM data with your feature flags to provide visibility into performance monitoring and behavioral changes. Determine which users are shown a user experience and if it is negatively affecting the user's performance.
```

Comment:
Can you add 1-2 sentences in the beginning on what Launchdarkly does?

Reply:
Typically we like to have the following in the overview: 

1. What does Launchdarkly do in 1-2 sentence. 
2. What will customers get from this integration? 
3. What types of data and the value of that data?